src/store/modules/keymap.js

```diff
@@ -374,6 +374,10 @@ const mutations = {
       }
     }
   },
+  duplicateLayer(state, { layer }) {
+    state.keymap = [...state.keymap, state.keymap[layer]];
```

Comment:
what happens if there are more than 16 layers?

Reply:
It just keeps going. I wasn't sure if 15 was actually the cap. Is there a global constant we have with the 15 cap?

Reply:
I would add it to `src/store/modules/constants.js`, or just define it within the `keymap.js`